repository-data/application/src/main/resources/hcm-config/namespaces/hee/blockLinks.yaml

```diff
@@ -0,0 +1,103 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/blockLinks:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 4953da19-ea58-461e-9eaf-ebe1085bc4e8
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: ec738eff-a3c8-4bef-bc80-9279e8b226cd
+        /hipposysedit:nodetype[1]:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['hipposysedit:remodel', 'mix:referenceable']
+          jcr:uuid: c91ecca8-5d82-4aba-9779-138df87eebac
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hee:basedocument', 'hippostd:relaxed', 'hippotranslation:translated']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /title:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:title
+            hipposysedit:primary: false
+            hipposysedit:type: Text
+          /description:
```

Comment:
Could we change `description` as a `text` type field with `TextArea` editor (in order to be inline with other cards/blocks) ?

Reply:
Yes. Commited on https://github.com/Manifesto-Digital/hee-cms-platform/pull/82/commits/548764f8d548337f9cd8adb6b003d052bebefc98